README.md

```diff
@@ -3,6 +3,7 @@ Puppet Catalog Diff Viewer
 
 [![License](https://img.shields.io/github/license/voxpupuli/puppet-catalog-diff-viewer.svg)](https://github.com/voxpupuli/puppet-catalog-diff-viewer/blob/master/LICENSE)
 [![CI](https://github.com/voxpupuli/puppet-catalog-diff-viewer/actions/workflows/ci.yaml/badge.svg)](https://github.com/voxpupuli/puppet-catalog-diff-viewer/actions/workflows/ci.yaml)
+[![Donated by Camptocamp](https://img.shields.io/badge/donated%20by-camptocamp-fb7047.svg)](#transfer-notice)
```

Comment:
please also add the `Transfer notice` section. in other repos we have:

```
## Transfer Notice

This project was originally authored by [Camptocamp](http://www.camptocamp.com).
The maintainer preferred that Vox Pupuli take ownership of the project for future improvement and maintenance.
Existing pull requests and issues were transferred over, please fork and continue to contribute here instead of Camptocamp.
```

Reply:
as you please oh dear great remote executor!